src/app/states/editing-note/page.mdx

```diff
@@ -0,0 +1,44 @@
+export const metadata = {
+  title: 'editingNote',
+  description: 'An app state of editingNote in Inkdrop. This section explains how to fetch the `editingNote` state.',
+}
+
+# editingNote
+
+An [app state](/guides/flux-architecture) of editingNote in Inkdrop.
+You can obtain the `books` state and integrate it with React components.{{ className: 'lead' }}
+
+
+---
+
+## Data Structure
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="all" type="Book[]">
+        The state represents [Note][note] data while a note opened, otherwise it's `null` or `undefined`.
+      </Property>
+    </Properties>
+  </Col>
```

Comment:
This should only be a reference to the note model, not an individual object structure.
```suggestion
  <Col>
    The state represents [Note][note] data while a note opened, otherwise it's `null` or `undefined`.
  </Col>
```